### DISCUSSION TITLE
Can not pass the pre-check

### DISCUSSION THREAD

**Discussion by Doeca** [community/community#35709] [GitHub Education]
I am from China and use MacBook with safari and chrome.
I have tried both browsers to upload any of my certificates to prove my student status.
But the pre-check always failed for the messages followed:
1. Please upload a different document to demonstrate your current academic status.
2. Modify your browser settings to allow GitHub to use your location information. Once location is allowed you MUST cancel your current application and begin again.
3. You are significantly more likely to be verified if you have completed your GitHub user profile with your full name and a short bio.

And sometimes there are even other errors such as "There is no campus you selected in your location".

I can even FaceTime with the Manual customer service to prove my student identity but there isn't any of that.

I am going to be crazy about this robotic system.

**Comment by SCHsebastian** [comment]
Having same ISUE, I've changed 3 times of navigator, accepted EVERYTHING of github to get my data, but it doent's seem to be ok for them. IT DOESNT WORK

**Comment by Lan-ce-lot** [comment]
I'm from China and use MacBook with chrome, too.
There is my failed for the messages:
```
Your application cannot be reviewed until you fix the following:
Modify your browser settings to allow GitHub to use your location information. Once location is allowed you MUST cancel your current application and begin again.
You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name and a short bio.
Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?
```
So it depends on...

**Comment by seu0948** [comment]
the only work-around seems to be getting $hit from 闲鱼 at the cost of around 5-10 bucks, mostly likely microsoft dont give a $hit about what you write here

**Comment by Lan-ce-lot** [comment]
> I am from China and use MacBook with safari and chrome. I have tried both browsers to upload any of my certificates to prove my student status. But the pre-check always failed for the messages followed:
> 
> 1. Please upload a different document to demonstrate your current academic status.
> 2. Modify your browser settings to allow GitHub to use your location information. Once location is allowed you MUST cancel your current application and begin again.
> 3. You are significantly more likely to be verified if you have completed your GitHub user profile with your full name and a short bio.
> 
> And sometimes there are even other errors such as "There is no campus you selected in your location".
> 
> I can even FaceTime with the Manual customer service to prove my student identity but there isn't any of that.
> 
> I am going to be crazy about this robotic system.

I think this is the problem with the macOS Chrome browser:

When I change my browser to Firefox, I can get rid of the first location failed message

![image](https://user-images.githubusercontent.com/53330071/199433136-b86d324b-fdc6-4a61-8006-e42a9b4dfbd6.png)


Good Luck
